components/Map/index.jsx

```diff
@@ -2,18 +2,18 @@ import { MapContainer, TileLayer, LayersControl } from 'react-leaflet';
 import Marker from '~/components/Marker';
 
 const tiles = [
-  {
-    name: 'Earth',
-    url: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
-    checked: true
-  },
   {
     name: 'Light',
-    url: 'https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}{r}.png'
+    url: 'https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}{r}.png',
+    checked: true
   },
   {
     name: 'Dark',
     url: 'https://tiles.stadiamaps.com/tiles/alidade_smooth_dark/{z}/{x}/{y}{r}.png'
+  },
+  {
+    name: 'Earth',
+    url: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
```

Comment:
I think we should make the Earth as default to save some requests to the Mapbox API. We could change it for users that already made a preference, save it to localStorage.

Reply:
For now I'm not sure how to do that, so I'll just make it Earth by default again ;)

Reply:
I started to do it on PR

Reply:
Great! I was taking a look at it. Could you approve this one though? I kinda wanted to make another PR implementing the ability to filter the pins by type, I already tried it out and its working. I shared a screenshot in the CeSIUM Discord :)